packages/website/src/pages/foundations/SvgExamples.tsx

```diff
@@ -4,12 +4,11 @@ import {
     IItemBoxProps,
     PlasmaState,
     Section,
-    SingleSelectConnected,
+    SingleSelectWithFilter,
     StringUtils,
     Svg,
-    SVG_NAMES,
-    SvgNames,
 } from '@coveord/plasma-react';
+import {svg as Icons, SvgName} from '@coveord/plasma-style';

```

Comment:
I guess we do svg as Icon because just naming it icons directly could be misleading? :thinking: 
We have svgs that are not icons right?

Reply:
Its mostly because I didn't want to rename `svg` since its used in a lot of places including old backbone code and I didn't want to go change that twice considering the icons extraction is coming.